app.css

```diff
@@ -304,16 +304,26 @@ styling the header component
     display: none;
     position: fixed;
     overflow-y: scroll;
-    left: 16%;
-    right: auto;
+    left: 1px;
     top: 1px;
-    width: 920px;
-    height: 1083px;
+    width: 1440px;
+    height: 1990px;
     z-index: 2;
+    padding: 120px 260px;
+    background: #c1c7d0;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] Kindly notice that the project card in the desktop version is not working well you need to fix it, here I gave you the code to make look like Figma.

![Capture6](https://user-images.githubusercontent.com/33752735/188997127-59296f45-5412-4454-9571-52f885535cb5.PNG)

```css
#project1amend1desc2 {
    width: 920px;
    padding: 16px;
    background-color: #fff;
    position: relative;
    margin: auto;
    height: 1083px;
    border-radius: 8px;
    border: 1px solid #ebecf0;
    box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
  }
```